ONSdigital/ssdc-rm-support-tool
# 2932 Validate email address format in create user screen

# Motivation and Context
Invalid email addresses can never be valid users, so we should validate the email address format in the create user dialogue.

# What has changed
- Add email address format validation

# How to test?
Build and run, try to create user with anything that is not in a valid email address format, it should be rejected.

# Links
https://trello.com/c/4OKxHZWr/2932-support-tool-user-email-input-is-not-validated
